src/lib.rs

```diff
@@ -46,6 +73,13 @@ impl slog::Drain for TracingSlogDrain {
                 return;
             }
 
+            #[cfg(feature = "kv")]
+            let field_serializer = {
+                let mut ser = TracingKvSerializer::default();
+                let _ = record.kv().serialize(record, &mut ser);
```

Comment:
Should we return an error here, or not worth it?

Reply:
Hm, I'm not sure it's worth it here since as a user of the log, I'd rather have *some* data with missing kv-fields than nothing.